app/controllers/attributes_controller.rb

```diff
@@ -28,11 +35,19 @@ def update
 
 private
 
-  def get_attributes_from_params(attributes, is_hash: false)
+  def get_attributes_from_params(attributes, permission_level:, is_hash: false)
```

Comment:
Permission levels must be either `:get` `:set` or `:check` right?
Is it work putting any kind fo guard clause up here with an error message for an invalid permission_level?

I guess that would be caught with `ApiError::LevelOfAuthenticationTooLow` if i passed this `:update` or something?

Reply:
It'll throw an error because of the fetch in

```
  def level_of_authentication_for(name, permission_level)
    attributes.fetch(name)[:permissions].fetch(permission_level)
  end
```